### DISCUSSION TITLE
ImageMagick-Windows still refers to VisualMagick but does not clone it's repo?

### DISCUSSION THREAD

**Discussion by cfitz** [ImageMagick/ImageMagick#7221] [General] [answered]
Hi,

I am needing to build ImageMagick for Windows with FFTW. I have done this previously following the README in the ImageMagick-Windows repo, which has a CloneRepositories script. This used to clone the VisualMagick repo, which had a wizard to configure a VS solution. 
However, this no longer clones the VisualMagick and I noticed that VM repo is now read-only since a few days ago. 
I assume VisualMagick repo is no longer the way to go, but the README in IM-Windows still refers to VisualMagick. 

So, what is the best option to build IM with FFTW and HDRI for Windows? I need the subimage search speed improvements. 
I am assuming there's not a binary distro floating around. 

Thanks!

**Comment by dlemstra** [comment, marked as answer]
The `VisualMagick` repository has been archived. I just updated the readme of the `ImageMagick-Windows` repository with the new instructions.

**Reply by cfitz** [reply]
Thanks!
